### DISCUSSION TITLE
HIVE - PYODBC - MYSQL?

### DISCUSSION THREAD

**Discussion by patryk-suwala** [mkleehammer/pyodbc#1049] [Q&A] [answered]
I am connected to Databricks/Hive Database via PYODBC but I need some functions that are not available in HiveSQL like TIMEDIFF(). What is the best course of action here?

**Comment by v-chojas** [comment, marked as answer]
pyODBC doesn't do anything with the query but passes it through to the underlying ODBC driver. As such, this isn't a question for pyODBC, but instead you should contact the driver and/or database vendor.
